ControllerKey.js

```diff
@@ -19,30 +19,30 @@ const _seedCache = new SeedCache();
 
 export class ControllerKey {
   /**
-   * Creates a new instance of a ControllerKey. This function should never
-   * be called directly. Use one of these methods to create a ControllerKey
-   * instance.
-   *
+   * Creates a new instance of a ControllerKey.
    * A ControllerKey can be passed as an `invocationSigner` to a KmsClient, but
    * a KmsClient instance is typically used internally by other instances that
    * can be created via the ControllerKey API such as instances of the Kek
    * and Hmac classes.
    *
+   * This function should never be called directly.
+   * Use one of these methods to create a ControllerKey instance.
```

Comment:
Merge into a single paragraph or add whitespace to separate paragraphs here and above.

Reply:
The rule is: Paragraphs should have an empty whitespace line after them (even if they are a single sentence).

Reply:
Addressed here, but I added one line of test and made it a single paragraph: https://github.com/digitalbazaar/web-kms-client/pull/2/commits/ba4f52d638724b1761a012e4a1290affd65df190